typings/index.d.ts

```diff
@@ -446,8 +446,12 @@ export class CommandInteractionOptionResolver {
 
   public get(name: string, required: true): CommandInteractionOption;
   public get(name: string, required?: boolean): CommandInteractionOption | null;
-  public getSubCommand(): string;
-  public getSubCommandGroup(): string;
+
+  public getSubCommand(required?: true): string;
+  public getSubCommand(required: boolean): string | null;
```

Comment:
This overload will never be matched iirc because its below the one with possible undefined. You should make an overload where required is false, then one for `required?: boolean`

Reply:
It matches in `index.ts` though